### PR TITLE
Cleanup for 2.0.0 Release

### PR DESCRIPTION
**Pull request by yagudaev**
This is still a work-in-progress and started as a bit of a spike to see if I can fix the security concerns with allowing arbitrary code execution. #13 

I'm adding this PR here as a way to start a conversation about the direction we want for a 2.0.0 release and outline what we as a community want to see there. 

# Changes

* Remove `Cypress` and `CypressDev` namespaces and consolidate things under `CypressOnRails`
* Style updates to the readme 
* Add logging access
* Fix destroy action to undo what the generator created -- still needs more robust tests

I marked this as an Alpha since my team is actually using this for our testing with cypress.

Now that I think about it more, would you be open to create a 2.0.0 branch and a tag for issues we want to see on a 2.0.0 release?  The main reason this has to be in a major version release is that it breaks backward compatibility to make things cleaner. We can even add deprecation messages to next minor releases to encourage existing users to change.

Thoughts?

### CODE REVIEW

**Comment by grantspeelman** [comment]
i think the only major thing is the renaming `CypressDev` to `CypressOnRails`.
Let me see if I can pull that from this PR and I'll see if we can keep backward compatibility then we don't have to do a major version release as essentially it still does the same thing.
Once we have this in then we chat about the other changes on a case by case basis.

**Comment by yagudaev** [comment]
@grantspeelman you are right, is there a reason to keep the `CypressDev` alive? If the official name is now `CypressOnRails` does this matter?

**Comment by grantspeelman** [comment]
@yagudaev I don't see any reason to rush the remove of `CypressDev`, will add the warnings about it though.

File: lib/cypress-on-rails.rb [hunk 1]
```diff
@@ -1,3 +1 @@
-require 'cypress_dev'
-
-Cypress = CypressDev
```
Comment (yagudaev) [inline lib/cypress-on-rails.rb]: @grantspeelman would it be enough to have 

```ruby
CypressDev = CypressOnRails
```

In here to maintain backward compatibility?

Comment (yagudaev) [reply lib/cypress-on-rails.rb]: nvm, saw #19 it should pave the way for this and I can then cleanup and rebase this PR

**Comment by justin808** [comment]
Hi @grantspeelman and @yagudaev, where do we stand with this PR? Should we close it? or update to master? @yagudaev are you using the code from this branch in production?

**Comment by grantspeelman** [comment]
The rename has already been done on master. 
@yagudaev you happy to close this branch?

**Comment by justin808** [comment]
@yagudaev anything we should merge here? Or close this?

**Comment by yagudaev** [comment]
> @yagudaev anything we should merge here? Or close this?

@justin808 and @grantspeelman 

I'm closing this for now. I'd like to move our code base at work to using origin again. You guys have done a fantastic job. We have also learned a lot since we started using Cypress.

**Pull request closed without merging**